app/models/queue_item.rb

```diff
@@ -0,0 +1,18 @@
+class QueueItem < ActiveRecord::Base
+
+  validates_presence_of :user_id, :video_id
```

Comment:
You can, I just wanted to make sure you knew that you didn't have to. Ruby is intended to be very expressive, while reading like english. Rails magic provides you the ability to validate the object itself instead of the actual column. There is nothing wrong with doing it this way though :smile: